web/src/app/components/loi-panel-header/loi-panel-header.component.ts

```diff
@@ -39,7 +39,8 @@ import { Point } from '../../shared/models/geometry/point';
   styleUrls: ['./loi-panel-header.component.scss'],
 })
 export class LocationOfInterestPanelHeaderComponent
-  implements OnInit, OnDestroy, OnChanges {
+  implements OnInit, OnDestroy, OnChanges
+{
```

Comment:
🤔 Very strange syntax but okay, I guess we are using `ng lint` now.

Reply:
Perhaps it did this because the declaration is split on multiple lines? If what I did is WAI, we're now applying the gts eslint configs via ng lint. Before this change they weren't being enforced.

Reply:
I see, it's good we have the lint forced.